variables.js

```diff
@@ -15,6 +15,14 @@
     2. Use console.log() to output the value of each variable.
  ********************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let personName = "ali";
+
+console.log(personName);
+let age = "15";
```

Comment:
When declaring a variable that is meant to hold a numeric value, such as age in this case, you should assign it a numerical value directly without enclosing it in double quotation marks. Here's the corrected version of the code:
**let age = 15;**